gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
```

Comment:
evt al naar 8.1

Reply:
Heb ik nog niet gedaan, mijn ervaring is vaak dat de allernieuwste versie van Gradle soms ruzie heeft met Spring dus vandaar